webapp/channels/src/sass/routes/_settings.scss

```diff
@@ -198,16 +198,6 @@
                 vertical-align: top;
             }
 
-            .nav {
-                position: fixed;
```

Comment:
@asaadmahmood we might still need this. If the content of the modal (i.e. right side of the modal) is scrollable, navigation also scrolls. If you agree, I can add back on my PR